README.md

```diff
@@ -184,6 +184,24 @@ dangerousDisableContainerSupport: true
 Alternatively, the presence of ``-XX:MaxRAM=`` prefix in either static or custom jvm opts will also disable this
 behavior.
 
+### Graal Native Image support
+
+Go-java-launcher supports running a native image of a Java application. For example:
```

Comment:
Maybe worth including a comment explaining that this is targeting native images built by graal-native? (considering some of the assumptions we're making regarding what these would support vs not)